src/main/steps/tasklistresponse/proceedings/documentUpload/template.njk

```diff
@@ -11,7 +11,7 @@
 
 {% if FileErrors.length > 0 %}
 {{ govukErrorSummary({
-  titleText: "There is a problem",
+  titleText: {{problem}},
```

Comment:
I don't think this will work. Test and let me know

Reply:
Yes Vivek , Done a small change .. its working fine now. Tested that change too.